### ISSUE TITLE
[package] sdl/2.0.20: fails to find wayland-scanner

### ISSUE THREAD

**Issue by andoalon** [conan-io/conan-center-index#11889]
### Package and Environment Details

* Package Name/Version: **sdl/2.0.20**
* Operating System+version: **Ubuntu 22.04 LTS (WSL2 Windows 10)**
* Compiler+version: **GCC 11.2.0**
* Conan version: **conan 1.50.0**
* Python version: **Python 3.10.4**

### Conan profile
```
[settings]
arch=x86_64
arch_build=x86_64
build_type=Debug
compiler=gcc
compiler.cppstd=17
compiler.libcxx=libstdc++11
compiler.version=11
os=Linux
os_build=Linux
[options]
[build_requires]
[env]
```

### Steps to reproduce

Install sdl/2.0.20:
> Note: I'm disabling some options because I get other issues with those libraries but I don't really need them for now

`conan install sdl/2.0.20@ --profile=my-profile --build=missing -o sdl:alsa=False -o sdl:nas=False -o sdl:jack=False -o sdl:pulse=False`

See error

### Logs

<details><summary>Click to expand log</summary>

```
sdl/2.0.20: Applying build-requirement: pkgconf/1.7.4
sdl/2.0.20: WARN: Build folder is dirty, removing it: /home/andoalon/.conan/data/sdl/2.0.20/_/_/build/0f97652040d5e24ccd00e2a95337264185234eba
sdl/2.0.20: Copying sources to build folder
sdl/2.0.20: Building your package in /home/andoalon/.conan/data/sdl/2.0.20/_/_/build/0f97652040d5e24ccd00e2a95337264185234eba
sdl/2.0.20: Generator pkg_config created pkgconf.pc
sdl/2.0.20: Generator pkg_config created opengl.pc
sdl/2.0.20: Generator pkg_config created xkbcli-interactive-wayland.pc
sdl/2.0.20: Generator pkg_config created xkbregistry.pc
sdl/2.0.20: Generator pkg_config created xkbcommon.pc
sdl/2.0.20: Generator pkg_config created xkbcommon-x11.pc
sdl/2.0.20: Generator pkg_config created xkbcommon_all_do_not_use.pc
sdl/2.0.20: Generator pkg_config created egl.pc
sdl/2.0.20: Generator pkg_config created libunwind.pc
sdl/2.0.20: Generator pkg_config created libunwind-generic.pc
sdl/2.0.20: Generator pkg_config created libunwind-coredump.pc
sdl/2.0.20: Generator pkg_config created libunwind-setjmp.pc
sdl/2.0.20: Generator pkg_config created libunwind-ptrace.pc
sdl/2.0.20: Generator pkg_config created wayland-egl-backend.pc
sdl/2.0.20: Generator pkg_config created wayland-client.pc
sdl/2.0.20: Generator pkg_config created wayland-egl.pc
sdl/2.0.20: Generator pkg_config created wayland-cursor.pc
sdl/2.0.20: Generator pkg_config created wayland-server.pc
sdl/2.0.20: Generator pkg_config created wayland-scanner.pc
sdl/2.0.20: Generator pkg_config created wayland.pc
sdl/2.0.20: Generator pkg_config created uuid.pc
sdl/2.0.20: Generator pkg_config created xcb-dri3.pc
sdl/2.0.20: Generator pkg_config created xcb-util.pc
sdl/2.0.20: Generator pkg_config created xcb-event.pc
sdl/2.0.20: Generator pkg_config created xcb-aux.pc
sdl/2.0.20: Generator pkg_config created xcb-atom.pc
sdl/2.0.20: Generator pkg_config created xkeyboard-config.pc
sdl/2.0.20: Generator pkg_config created xcb.pc
sdl/2.0.20: Generator pkg_config created xcb-xinerama.pc
sdl/2.0.20: Generator pkg_config created xcb-xfixes.pc
sdl/2.0.20: Generator pkg_config created xcb-sync.pc
sdl/2.0.20: Generator pkg_config created xcb-shm.pc
sdl/2.0.20: Generator pkg_config created xcb-shape.pc
sdl/2.0.20: Generator pkg_config created xcb-renderutil.pc
sdl/2.0.20: Generator pkg_config created xcb-render.pc
sdl/2.0.20: Generator pkg_config created xcb-randr.pc
sdl/2.0.20: Generator pkg_config created xcb-keysyms.pc
sdl/2.0.20: Generator pkg_config created xcb-image.pc
sdl/2.0.20: Generator pkg_config created xcb-icccm.pc
sdl/2.0.20: Generator pkg_config created xcb-xkb.pc
sdl/2.0.20: Generator pkg_config created xtrans.pc
sdl/2.0.20: Generator pkg_config created xxf86vm.pc
sdl/2.0.20: Generator pkg_config created xvmc.pc
sdl/2.0.20: Generator pkg_config created xv.pc
sdl/2.0.20: Generator pkg_config created xtst.pc
sdl/2.0.20: Generator pkg_config created xt.pc
sdl/2.0.20: Generator pkg_config created xscrnsaver.pc
sdl/2.0.20: Generator pkg_config created xres.pc
sdl/2.0.20: Generator pkg_config created xrender.pc
sdl/2.0.20: Generator pkg_config created xrandr.pc
sdl/2.0.20: Generator pkg_config created xpm.pc
sdl/2.0.20: Generator pkg_config created xmuu.pc
sdl/2.0.20: Generator pkg_config created xmu.pc
sdl/2.0.20: Generator pkg_config created xkbfile.pc
sdl/2.0.20: Generator pkg_config created xinerama.pc
sdl/2.0.20: Generator pkg_config created xi.pc
sdl/2.0.20: Generator pkg_config created xfixes.pc
sdl/2.0.20: Generator pkg_config created xext.pc
sdl/2.0.20: Generator pkg_config created xdmcp.pc
sdl/2.0.20: Generator pkg_config created xdamage.pc
sdl/2.0.20: Generator pkg_config created xcursor.pc
sdl/2.0.20: Generator pkg_config created xcomposite.pc
sdl/2.0.20: Generator pkg_config created xaw7.pc
sdl/2.0.20: Generator pkg_config created xau.pc
sdl/2.0.20: Generator pkg_config created sm.pc
sdl/2.0.20: Generator pkg_config created ice.pc
sdl/2.0.20: Generator pkg_config created fontenc.pc
sdl/2.0.20: Generator pkg_config created x11-xcb.pc
sdl/2.0.20: Generator pkg_config created x11.pc
sdl/2.0.20: Generator pkg_config created xorg.pc
sdl/2.0.20: Generator pkg_config created wayland-protocols.pc
sdl/2.0.20: Generator pkg_config created liblzma.pc
sdl/2.0.20: Generator pkg_config created libffi.pc
sdl/2.0.20: Generator pkg_config created libxml-2.0.pc
sdl/2.0.20: Generator pkg_config created expat.pc
sdl/2.0.20: Generator pkg_config created libiconv.pc
sdl/2.0.20: Generator pkg_config created zlib.pc
sdl/2.0.20: Generator cmake created conanbuildinfo.cmake
sdl/2.0.20: Aggregating env generators
sdl/2.0.20: Calling build()
-- The C compiler identification is GNU 11.2.0
-- The CXX compiler identification is GNU 11.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting language standard
-- Conan: Adjusting fPIC flag (ON)
-- Conan: Compiler GCC>=5, checking major version 11
-- Conan: Checking correct version: 11
-- Found PkgConfig: /home/andoalon/.conan/data/pkgconf/1.7.4/_/_/package/cc9f9b2b9f92fc82888026414560bf79d8c3a052/bin/pkgconf (found version "1.7.4")
-- Performing Test HAVE_GCC_WALL
-- Performing Test HAVE_GCC_WALL - Success
-- Performing Test HAVE_GCC_NO_STRICT_ALIASING
-- Performing Test HAVE_GCC_NO_STRICT_ALIASING - Success
-- Performing Test HAVE_GCC_WDECLARATION_AFTER_STATEMENT
-- Performing Test HAVE_GCC_WDECLARATION_AFTER_STATEMENT - Success
-- Performing Test HAVE_GCC_WERROR_DECLARATION_AFTER_STATEMENT
-- Performing Test HAVE_GCC_WERROR_DECLARATION_AFTER_STATEMENT - Success
-- Performing Test HAVE_GCC_ATOMICS
-- Performing Test HAVE_GCC_ATOMICS - Success
-- Performing Test HAVE_GCC_PREFERRED_STACK_BOUNDARY
-- Performing Test HAVE_GCC_PREFERRED_STACK_BOUNDARY - Failed
-- Performing Test HAVE_GCC_FVISIBILITY
-- Performing Test HAVE_GCC_FVISIBILITY - Success
-- Performing Test HAVE_GCC_WSHADOW
-- Performing Test HAVE_GCC_WSHADOW - Success
-- Performing Test HAVE_NO_UNDEFINED
-- Performing Test HAVE_NO_UNDEFINED - Success
-- Performing Test HAVE_MMX
-- Performing Test HAVE_MMX - Success
-- Performing Test HAVE_3DNOW
-- Performing Test HAVE_3DNOW - Success
-- Performing Test HAVE_SSE
-- Performing Test HAVE_SSE - Success
-- Performing Test HAVE_SSE2
-- Performing Test HAVE_SSE2 - Success
-- Performing Test HAVE_SSE3
-- Performing Test HAVE_SSE3 - Success
-- Looking for immintrin.h
-- Looking for immintrin.h - found
-- Performing Test HAVE_ALTIVEC_H_HDR
-- Performing Test HAVE_ALTIVEC_H_HDR - Failed
-- Performing Test HAVE_ALTIVEC
-- Performing Test HAVE_ALTIVEC - Failed
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdio.h
-- Looking for stdio.h - found
-- Looking for stdlib.h
-- Looking for stdlib.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Looking for stdarg.h
-- Looking for stdarg.h - found
-- Looking for malloc.h
-- Looking for malloc.h - found
-- Looking for memory.h
-- Looking for memory.h - found
-- Looking for string.h
-- Looking for string.h - found
-- Looking for limits.h
-- Looking for limits.h - found
-- Looking for float.h
-- Looking for float.h - found
-- Looking for strings.h
-- Looking for strings.h - found
-- Looking for wchar.h
-- Looking for wchar.h - found
-- Looking for inttypes.h
-- Looking for inttypes.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for ctype.h
-- Looking for ctype.h - found
-- Looking for math.h
-- Looking for math.h - found
-- Looking for iconv.h
-- Looking for iconv.h - found
-- Looking for signal.h
-- Looking for signal.h - found
-- Looking for 7 include files stddef.h, ..., float.h
-- Looking for 7 include files stddef.h, ..., float.h - found
-- Check size of size_t
-- Check size of size_t - done
-- Looking for M_PI
-- Looking for M_PI - found
-- Performing Test HAVE_MPROTECT
-- Performing Test HAVE_MPROTECT - Success
-- Looking for strtod
-- Looking for strtod - found
-- Looking for malloc
-- Looking for malloc - found
-- Looking for calloc
-- Looking for calloc - found
-- Looking for realloc
-- Looking for realloc - found
-- Looking for free
-- Looking for free - found
-- Looking for getenv
-- Looking for getenv - found
-- Looking for setenv
-- Looking for setenv - found
-- Looking for putenv
-- Looking for putenv - found
-- Looking for unsetenv
-- Looking for unsetenv - found
-- Looking for qsort
-- Looking for qsort - found
-- Looking for abs
-- Looking for abs - found
-- Looking for bcopy
-- Looking for bcopy - found
-- Looking for memset
-- Looking for memset - found
-- Looking for memcpy
-- Looking for memcpy - found
-- Looking for memmove
-- Looking for memmove - found
-- Looking for memcmp
-- Looking for memcmp - found
-- Looking for strlen
-- Looking for strlen - found
-- Looking for strlcpy
-- Looking for strlcpy - not found
-- Looking for strlcat
-- Looking for strlcat - not found
-- Looking for _strrev
-- Looking for _strrev - not found
-- Looking for _strupr
-- Looking for _strupr - not found
-- Looking for _strlwr
-- Looking for _strlwr - not found
-- Looking for index
-- Looking for index - found
-- Looking for rindex
-- Looking for rindex - found
-- Looking for strchr
-- Looking for strchr - found
-- Looking for strrchr
-- Looking for strrchr - found
-- Looking for strstr
-- Looking for strstr - found
-- Looking for strtok_r
-- Looking for strtok_r - found
-- Looking for itoa
-- Looking for itoa - not found
-- Looking for _ltoa
-- Looking for _ltoa - not found
-- Looking for _uitoa
-- Looking for _uitoa - not found
-- Looking for _ultoa
-- Looking for _ultoa - not found
-- Looking for strtol
-- Looking for strtol - found
-- Looking for strtoul
-- Looking for strtoul - found
-- Looking for _i64toa
-- Looking for _i64toa - not found
-- Looking for _ui64toa
-- Looking for _ui64toa - not found
-- Looking for strtoll
-- Looking for strtoll - found
-- Looking for strtoull
-- Looking for strtoull - found
-- Looking for atoi
-- Looking for atoi - found
-- Looking for atof
-- Looking for atof - found
-- Looking for strcmp
-- Looking for strcmp - found
-- Looking for strncmp
-- Looking for strncmp - found
-- Looking for _stricmp
-- Looking for _stricmp - not found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for _strnicmp
-- Looking for _strnicmp - not found
-- Looking for strncasecmp
-- Looking for strncasecmp - found
-- Looking for wcscmp
-- Looking for wcscmp - found
-- Looking for _wcsdup
-- Looking for _wcsdup - not found
-- Looking for wcsdup
-- Looking for wcsdup - found
-- Looking for wcslcat
-- Looking for wcslcat - not found
-- Looking for wcslcpy
-- Looking for wcslcpy - not found
-- Looking for wcslen
-- Looking for wcslen - found
-- Looking for wcsncmp
-- Looking for wcsncmp - found
-- Looking for wcsstr
-- Looking for wcsstr - found
-- Looking for wcscasecmp
-- Looking for wcscasecmp - found
-- Looking for _wcsicmp
-- Looking for _wcsicmp - not found
-- Looking for wcsncasecmp
-- Looking for wcsncasecmp - found
-- Looking for _wcsnicmp
-- Looking for _wcsnicmp - not found
-- Looking for sscanf
-- Looking for sscanf - found
-- Looking for vsscanf
-- Looking for vsscanf - found
-- Looking for vsnprintf
-- Looking for vsnprintf - found
-- Looking for fopen64
-- Looking for fopen64 - not found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for fseeko64
-- Looking for fseeko64 - not found
-- Looking for _Exit
-- Looking for _Exit - found
-- Looking for sigaction
-- Looking for sigaction - found
-- Looking for setjmp
-- Looking for setjmp - found
-- Looking for nanosleep
-- Looking for nanosleep - found
-- Looking for sysconf
-- Looking for sysconf - found
-- Looking for sysctlbyname
-- Looking for sysctlbyname - not found
-- Looking for getauxval
-- Looking for getauxval - found
-- Looking for elf_aux_info
-- Looking for elf_aux_info - not found
-- Looking for poll
-- Looking for poll - found
-- Looking for pow in m
-- Looking for pow in m - found
-- Looking for atan
-- Looking for atan - found
-- Looking for atan2
-- Looking for atan2 - found
-- Looking for atanf
-- Looking for atanf - found
-- Looking for atan2f
-- Looking for atan2f - found
-- Looking for ceil
-- Looking for ceil - found
-- Looking for ceilf
-- Looking for ceilf - found
-- Looking for copysign
-- Looking for copysign - found
-- Looking for copysignf
-- Looking for copysignf - found
-- Looking for cos
-- Looking for cos - found
-- Looking for cosf
-- Looking for cosf - found
-- Looking for exp
-- Looking for exp - found
-- Looking for expf
-- Looking for expf - found
-- Looking for fabs
-- Looking for fabs - found
-- Looking for fabsf
-- Looking for fabsf - found
-- Looking for floor
-- Looking for floor - found
-- Looking for floorf
-- Looking for floorf - found
-- Looking for fmod
-- Looking for fmod - found
-- Looking for fmodf
-- Looking for fmodf - found
-- Looking for log
-- Looking for log - found
-- Looking for logf
-- Looking for logf - found
-- Looking for log10
-- Looking for log10 - found
-- Looking for log10f
-- Looking for log10f - found
-- Looking for lround
-- Looking for lround - found
-- Looking for lroundf
-- Looking for lroundf - found
-- Looking for pow
-- Looking for pow - found
-- Looking for powf
-- Looking for powf - found
-- Looking for round
-- Looking for round - found
-- Looking for roundf
-- Looking for roundf - found
-- Looking for scalbn
-- Looking for scalbn - found
-- Looking for scalbnf
-- Looking for scalbnf - found
-- Looking for sin
-- Looking for sin - found
-- Looking for sinf
-- Looking for sinf - found
-- Looking for sqrt
-- Looking for sqrt - found
-- Looking for sqrtf
-- Looking for sqrtf - found
-- Looking for tan
-- Looking for tan - found
-- Looking for tanf
-- Looking for tanf - found
-- Looking for acos
-- Looking for acos - found
-- Looking for acosf
-- Looking for acosf - found
-- Looking for asin
-- Looking for asin - found
-- Looking for asinf
-- Looking for asinf - found
-- Looking for trunc
-- Looking for trunc - found
-- Looking for truncf
-- Looking for truncf - found
-- Looking for iconv_open in iconv
-- Looking for iconv_open in iconv - found
-- Looking for alloca.h
-- Looking for alloca.h - found
-- Looking for alloca
-- Looking for alloca - found
-- Looking for alloca
-- Looking for alloca - found
-- Looking for alloca
-- Looking for alloca - not found
-- Performing Test HAVE_SA_SIGACTION
-- Performing Test HAVE_SA_SIGACTION - Success
-- Looking for dlopen
-- Looking for dlopen - found
-- Performing Test HAVE_O_CLOEXEC
-- Performing Test HAVE_O_CLOEXEC - Success
-- Performing Test OSS_FOUND
-- Performing Test OSS_FOUND - Success
-- Checking for module 'libpipewire-0.3>=0.3.20'
--   Package 'libpipewire-0.3', required by 'virtual:world', not found
-- Looking for samplerate.h
-- Looking for samplerate.h - not found
-- dynamic libX11 -> libX11.so.6
-- dynamic libXext -> libXext.so.6
-- dynamic libXcursor -> libXcursor.so.1
-- dynamic libXinerama -> libXinerama.so.1
-- dynamic libXi -> libXi.so.6
-- dynamic libXfixes -> libXfixes.so.3
-- dynamic libXrandr -> libXrandr.so.2
-- dynamic libXrender -> libXrender.so.1
-- dynamic libXss -> libXss.so.1
-- dynamic libXxf86vm -> libXxf86vm.so.1
-- Looking for X11/extensions/XInput2.h
-- Looking for X11/extensions/XInput2.h - found
-- Looking for X11/extensions/Xfixes.h
-- Looking for X11/extensions/Xfixes.h - found
-- Looking for X11/extensions/Xrender.h
-- Looking for X11/extensions/Xrender.h - found
-- Looking for include files X11/Xlib.h, X11/extensions/Xdbe.h
-- Looking for include files X11/Xlib.h, X11/extensions/Xdbe.h - found
-- Looking for shmat
-- Looking for shmat - found
-- Performing Test HAVE_XGENERICEVENT
-- Performing Test HAVE_XGENERICEVENT - Success
-- Looking for XkbKeycodeToKeysym
-- Looking for XkbKeycodeToKeysym - found
-- Performing Test HAVE_XINPUT2_MULTITOUCH
-- Performing Test HAVE_XINPUT2_MULTITOUCH - Success
-- Performing Test HAVE_XFIXES_H
-- Performing Test HAVE_XFIXES_H - Success
-- Checking for module 'egl'
--   Found egl, version system
-- Performing Test HAVE_OPENGL_EGL
-- Performing Test HAVE_OPENGL_EGL - Success
-- Checking for modules 'libdrm;gbm;egl'
--   Package 'libdrm', required by 'virtual:world', not found
--   Package 'gbm', required by 'virtual:world', not found
-- Performing Test HAVE_OPENGL_GLX
-- Performing Test HAVE_OPENGL_GLX - Success
-- Performing Test HAVE_OPENGL
-- Performing Test HAVE_OPENGL - Success
-- Performing Test HAVE_OPENGLES_V1
-- Performing Test HAVE_OPENGLES_V1 - Failed
-- Performing Test HAVE_OPENGLES_V2
-- Performing Test HAVE_OPENGLES_V2 - Success
-- Checking for modules 'wayland-client;wayland-egl;wayland-cursor;egl;xkbcommon>=0.5.0'
--   Found wayland-client, version 1.20.0
--   Found wayland-egl, version 18.1.0
--   Found wayland-cursor, version 1.20.0
--   Found egl, version system
--   Found xkbcommon, version 1.4.0
CMake Error at source_subfolder/cmake/sdlchecks.cmake:636 (find_program):
  Could not find WAYLAND_SCANNER using the following names: wayland-scanner
Call Stack (most recent call first):
  source_subfolder/CMakeLists.txt:1269 (CheckWayland)


-- Configuring incomplete, errors occurred!
See also "/home/andoalon/.conan/data/sdl/2.0.20/_/_/build/0f97652040d5e24ccd00e2a95337264185234eba/build_subfolder/CMakeFiles/CMakeOutput.log".
See also "/home/andoalon/.conan/data/sdl/2.0.20/_/_/build/0f97652040d5e24ccd00e2a95337264185234eba/build_subfolder/CMakeFiles/CMakeError.log".
sdl/2.0.20:
sdl/2.0.20: ERROR: Package '0f97652040d5e24ccd00e2a95337264185234eba' build failed
sdl/2.0.20: WARN: Build folder /home/andoalon/.conan/data/sdl/2.0.20/_/_/build/0f97652040d5e24ccd00e2a95337264185234eba
ERROR: sdl/2.0.20: Error in build() method, line 353
        cmake = self._configure_cmake()
while calling '_configure_cmake', line 346
        cmake.configure(build_dir=self._build_subfolder)
        ConanException: Error 1 while executing cd 'build_subfolder' && cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE="Debug" -DCONAN_IN_LOCAL_CACHE="ON" -DCONAN_COMPILER="gcc" -DCONAN_COMPILER_VERSION="11" -DCONAN_CXX_FLAGS="-m64" -DCONAN_SHARED_LINKER_FLAGS="-m64" -DCONAN_C_FLAGS="-m64" -DBUILD_SHARED_LIBS="OFF" -DCMAKE_INSTALL_PREFIX="/home/andoalon/.conan/data/sdl/2.0.20/_/_/package/0f97652040d5e24ccd00e2a95337264185234eba" -DCMAKE_INSTALL_BINDIR="bin" -DCMAKE_INSTALL_SBINDIR="bin" -DCMAKE_INSTALL_LIBEXECDIR="bin" -DCMAKE_INSTALL_LIBDIR="lib" -DCMAKE_INSTALL_INCLUDEDIR="include" -DCMAKE_INSTALL_OLDINCLUDEDIR="include" -DCMAKE_INSTALL_DATAROOTDIR="share" -DCONAN_CMAKE_POSITION_INDEPENDENT_CODE="ON" -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY="ON" -DCONAN_EXPORTED="1" -DCONAN_INSTALL_FOLDER="/home/andoalon/.conan/data/sdl/2.0.20/_/_/build/0f97652040d5e24ccd00e2a95337264185234eba" -DSDL_STATIC_PIC="True" -DSDL_SHARED="False" -DSDL_STATIC="True" -DSDL_OPENGL="True" -DSDL_OPENGLES="True" -DSDL_VULKAN="True" -DSDL_VIDEO_DRIVER_X11_SUPPORTS_GENERIC_EVENTS="1" -DSDL_ALSA="False" -DSDL_JACK="False" -DSDL_ESD="False" -DSDL_PULSEAUDIO="False" -DSDL_SNDIO="False" -DSDL_NAS="False" -DSDL_X11="True" -DHAVE_XEXT_H="True" -DSDL_X11_XCURSOR="True" -DHAVE_XCURSOR_H="True" -DSDL_X11_XINERAMA="True" -DHAVE_XINERAMA_H="True" -DSDL_X11_XINPUT="True" -DHAVE_XINPUT_H="True" -DSDL_X11_XRANDR="True" -DHAVE_XRANDR_H="True" -DSDL_X11_XSCRNSAVER="True" -DHAVE_XSS_H="True" -DSDL_X11_XSHAPE="True" -DHAVE_XSHAPE_H="True" -DSDL_X11_XVM="True" -DHAVE_XF86VM_H="True" -DSDL_WAYLAND="True" -DSDL_WAYLAND_SHARED="False" -DSDL_DIRECTFB="False" -DSDL_RPI="False" -DHAVE_LIBUNWIND_H="True" -DEXTRA_LDFLAGS="-L/home/andoalon/.conan/data/wayland/1.20.0/_/_/package/80005ee34d2869de5a771d2ae3ed205ad546e215/lib" -DCMAKE_REQUIRED_INCLUDES="" -Wno-dev '/home/andoalon/.conan/data/sdl/2.0.20/_/_/build/0f97652040d5e24ccd00e2a95337264185234eba'
```

</details>

**Comment by ericLemanissier** [comment]
I think it's because wayland needs to be add as a build_requirements of sdl too, not only requirements

**Comment by toge** [comment]
SDL already has build_require wayland.
https://github.com/conan-io/conan-center-index/blob/eb4c167c772cc4b7cd50effa52e29f84d2989afd/recipes/sdl/all/conanfile.py#L168-L172

I think this compilation error is caused by the modification of wayland recipe in 14th July.
Now wayland recipe uses VirtualBuildEnv and VirtualRunEnv instead of env_info.
https://github.com/conan-io/conan-center-index/commit/3ec47a3baa33981096ce621f87c8885cd37cdc07

I am going to fix this problem in this PR. https://github.com/conan-io/conan-center-index/pull/11817

**Comment by paulocoutinhox** [comment]
It is blocking us to use some packages, like sdl, sdl_mixer.

I open PRs for this libraries that i found problems, but without wayland fixes, it is impossible.

**Comment by uilianries** [comment]
Related to #11986

**Comment by ericLemanissier** [comment]
This is probably fixed with #11896
